src/webgpu/shader/validation/expression/call/builtin/pack2x16float.spec.ts

```diff
@@ -0,0 +1,100 @@
+const kFn = 'pack2x16float';
+export const description = `Validate ${kFn}`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../../../common/util/data_tables.js';
+import { kValue } from '../../../../../../webgpu/util/constants.js';
+import { f32, vec2 } from '../../../../../util/conversion.js';
+import { ShaderValidationTest } from '../../../shader_validation_test.js';
+
+import { validateConstOrOverrideBuiltinEval } from './const_override_validation.js';
+
+const kArgCases = {
+  good: '(vec2f())',
```

Comment:
Shall we add an abstract float case? Like
``` javascript
  good_vec2_af: '(vec2(0.0))',
```

Reply:
Done